### DISCUSSION TITLE
Getting Build Failed Error when we trying to implement Wallet through EXPO - 48

### DISCUSSION THREAD

**Discussion by rahulkoodles** [WalletConnect/walletconnect-monorepo#5162] [Developer Support]
Hi Team,

We have a project running on EXPO - 48, and we're seeking to integrate Wallet Connect functionality into it. However, as I attempt to implement Waghmi.js, Ether.js, and Ether5, I encounter issues. It seems I may not be following the correct approach or may be missing some dependencies. Thus, I kindly request your guidance and support in this matter. Could you please provide proper instructions and perhaps share some code snippets illustrating how to establish a connection with any of these libraries within the context of EXPO-48?

I experimented with EXPO - 50, but unfortunately, it caused the previously developed application to crash. Interestingly, when I created a new project using EXPO - 50, I managed to successfully connect the wallet. Hence, I'm uncertain about the correct path to follow to integrate Wallet Connect with EXPO-48. Any assistance, guidance, or provision of relevant code would be greatly appreciated.

**Comment by glitch-txs** [comment]
Hello, could you share a minimal reproducible example? What errors do you see on console?
